app/.env.example

```diff
@@ -28,6 +28,12 @@ CHES_CLIENT_ID=
 CHES_CLIENT_SECRET=
 CHES_API_URL=
 
+CHES_MOCK_PORT=
```

Comment:
you don't need any of these environment variables:

CHES_MOCK_PORT=
SMTP_USERNAME=
SMTP_PASSWORD=
SMPT_HOST_NAME=
SMPT_PORT=